### DISCUSSION TITLE
App shows up twice in chrome://inspect. Is this normal?

### DISCUSSION THREAD

**Discussion by frederikheld** [ionic-team/capacitor#7532] [Q&A]
Hey everyone,

I just started using `chrome://inspect` in Chrome browser to debug some WebView specific issues and I'm wondering why it shows up twice (see screenshot).

![screenshot-capacitor-chrome-inspect](https://github.com/ionic-team/capacitor/assets/11460463/5e147926-f3db-482c-85c1-f8aa693583cd)

The app runs on my Pixel 7 phone through Android Studio. I can tap into both instances through the `inspect` link and each of them will open in a DevTools window that shows what's going on on the screen and allows to interact. They both stay in sync with the phone screen and each other as well. Only one of them is showing the `pause` link but it does not seem to have any effect anyways.

So I'm wondering: is this normal?

I re-created the whole thing with a new Capacitor app and it turns out that this does not seem to be an issue with my rather large app that I've built for a couple of months (which could have all kinds of side-effects with Capacitor), but actually also happens with a fresh app.

I've created this app following the [official tutorial](https://capacitorjs.com/docs/android). The only extra step I did was to upgrade the build tools to 8.4.0 when Android Studio prompted me to do so on the first launch.

I'd be interested in your thoughts on that. Should I be worried or is this normal? Why does the app show up twice in Chrome inspect?

Cheers
Frederik

**Comment by jadejr** [comment]
I don't know if it's normal or not. I don't think it used to, but it has for me for a few months at least probably. I've been too busy to ask about it myself. I can say you're not the only one seeing it.  I didn't go through the trouble replicating it with a fresh app like you did which I'd need to do to actually file a bug report to ask about it

**Comment by peitschie** [comment]
What are you settings here at the top of that screen?

![image](https://github.com/ionic-team/capacitor/assets/1052079/41157c6c-9784-4806-a787-b716fd38f979)

I found this sometimes when I had Chrome also trying to discover things. Generally, these will both be unchecked, as you want discovery to happen via `adb devices --list`

**Reply by jadejr** [reply]
with them selected I can get up to 4 . It's 2 without them selected.
